modules/dnn/src/layers/resize_layer.cpp

```diff
@@ -48,6 +48,7 @@ class ResizeLayerImpl : public ResizeLayer
         CV_Check(interpolation, interpolation == "nearest" || interpolation == "opencv_linear" || interpolation == "bilinear", "");
 
         alignCorners = params.get<bool>("align_corners", false);
+        halfPixelCenters = params.get<bool>("half_pixel_centers", false);
```

Comment:
Can you please check if OpenCV's resize with INTER_LINEAR matches "half_pixel_centers" behavior? See `interpolation == "opencv_linear"` path

Reply:
I've checked, OpenCV's resize with INTER_LINEAR repeats behaviour of TF `resize_bilinear` with `align_corners=False, half_pixel_center=True`. Updated the code.